Providers/Scripts/2.4x-2.5x/Scripts/nxOMSPerfCounter.py

```diff
@@ -32,18 +35,25 @@ def init_paths(WorkspaceID):
     multi_homed = os.path.isdir(mh_conf_dir)
 
     if multi_homed:
-        LG().Log('INFO', 'OMSAgent is multi-homed and resource is updating workspace ' + WorkspaceID)
+        LG().Log('INFO', 'Workspace config is using multi-homing structure; ' \
```

Comment:
Same comment as I had for OMSPlugin dsc resource. I think this log message is confusing. it does not properly capture multi homed versus single workspace structure diffs. What are you trying to log about with this line?

Reply:
I am trying to log the fact that we are using multi-homing paths from this point forward in the resource. I am not trying to log whether one or more than one workspace is configured on the machine, since the resource is only concerned with a single workspace.

Do you have any objections to the string I chose: "Workspace config is using multi-homing structure; resource is updating workspace xxxx"?

Reply:
I would rather remove the word multi-homing completely if that is the case. we can try resource versions using history and agent versions.

Reply:
How is this: "Configuration is in a workspace-specific path; resource is updating workspace xxxx"

Reply:
Yeah, better, agree now. thanks.

Reply:
Another thing is the variable name "multi_homed" makes the code hard to read.

Reply:
Can you also add OMSPlugin related change here (about the log message and parameter name) or send a separate PR ? Let us have these resources changed in consistent way.

Reply:
Changed in nxOMSPerfCounter for this PR.

Updating the version of resources for only a messaging and variable-naming change is a little costly. After this PR/messaging is approved, I will submit a PR to change the messaging in nxOMSPlugin/nxOMSSyslog/nxOMSSudoCustomLog, so that the change will go out with the next larger update.